lib/ansible/modules/cloud/cloudscale/cloudscale_server.py

```diff
@@ -376,44 +355,39 @@ def create_server(self):
         self.info = self._transform_state(self._post('servers', data))
         self.wait_for_state(('running', ))
 
-
     def delete_server(self):
         self._delete('servers/%s' % self.info['uuid'])
         self.wait_for_state(('absent', ))
 
-
     def start_server(self):
         self._post('servers/%s/start' % self.info['uuid'])
         self.wait_for_state(('running', ))
 
-
     def stop_server(self):
         self._post('servers/%s/stop' % self.info['uuid'])
         self.wait_for_state(('stopped', ))
 
-
     def list_servers(self):
         return self._get('servers')
 
 
 def main():
     module = AnsibleModule(
-        argument_spec = dict(
-            state               = dict(default='running',
-                                       choices=ALLOWED_STATES),
-            name                = dict(),
-            uuid                = dict(),
-            flavor              = dict(),
-            image               = dict(),
-            volume_size_gb      = dict(type='int', default=10),
-            bulk_volume_size_gb = dict(type='int'),
-            ssh_keys            = dict(type='list'),
-            use_public_network  = dict(type='bool', default=True),
-            use_private_network = dict(type='bool', default=False),
-            use_ipv6            = dict(type='bool', default=True),
-            anti_affinity_with  = dict(),
-            user_data           = dict(),
-            api_token           = dict(no_log=True),
+        argument_spec=dict(
+            state=dict(default='running', choices=ALLOWED_STATES),
+            name=dict(),
+            uuid=dict(),
+            flavor=dict(),
+            image=dict(),
+            volume_size_gb=dict(type='int', default=10),
+            bulk_volume_size_gb=dict(type='int'),
+            ssh_keys=dict(type='list'),
+            use_public_network=dict(type='bool', default=True),
+            use_private_network=dict(type='bool', default=False),
+            use_ipv6=dict(type='bool', default=True),
+            anti_affinity_with=dict(),
+            user_data=dict(),
+            api_token=dict(no_log=True),
```

Comment:
While agree to most of the changes this one has very dubious value IMO. It certainly does not improve readability.

Reply:
you mean "no_log=True"? This was already used and prevents secrets being logged.

Reply:
No I mean this whole chunk of the diff. Unfortunately github is not really clever in displaying this. My opinion is that aligning the = makes the code much more readable than strict adherence to PEP8.